perma_web/static/js/stats.js

```diff
@@ -1,7 +1,24 @@
 // House our sum data here. We should refactor this.
+window.StatsModule = window.StatsModule || {};
```

Comment:
Refraining from refactoring whole stats page for now, but new code follows the same style of other js modules